common/djangoapps/student/models.py

```diff
@@ -3461,6 +3463,27 @@ def should_celebrate_first_section(enrollment):
         except CourseEnrollmentCelebration.DoesNotExist:
             return False
 
+    @staticmethod
+    def should_celebrate_weekly_goal(enrollment):
```

Comment:
One difference between this and the other `should_` method in this class is that the other one just says whether the model says it should - that gets sent to the MFE that then uses other info to get the final answer.

Whereas this `should_` checks this model and also uses other info to get the final answer all in one call. Which is fine. But myabe draw a stronger distinction in their docstrings? Or add extra warnings to the other method that the answer isn't final, which maybe I didn't bother with.

Reply:
Yeah, I will add more information into both docstrings. Just the unfortunate outcome of my `should_` requiring information stored in the backend and the other `should_` relying on information only in the frontend.